datasets/GazeBase/stimulus_TEX_R1S1_bg/trial_1_participant_100_TEX_R1S1_bg.json

```diff
@@ -1,5 +1,356 @@
+        30202.0,
+        30202.0,
+        30423.0,
+        30423.0,
+        30696.0,
+        30907.0,
+        30907.0,
+        31151.0,
+        31326.0,
+        31326.0,
+        31557.0,
+        31557.0,
+        31770.0,
+        31988.0,
+        32180.0,
+        32277.0,
+        32647.0,
+        32706.0,
+        32928.0,
+        33154.0,
+        33393.0,
+        33393.0,
+        33665.0,
+        33899.0,
+        33899.0,
+        34105.0,
+        34308.0,
+        34515.0,
+        34642.0,
+        34642.0,
+        34733.0,
+        34733.0,
+        34885.0,
+        34885.0,
+        35337.0,
+        35521.0,
+        35521.0,
+        35521.0,
+        35624.0,
+        35624.0,
+        35624.0,
+        35890.0,
+        35890.0,
+        35890.0,
+        36054.0,
+        36054.0,
+        36190.0,
+        36248.0,
+        36492.0,
+        36492.0,
+        36754.0,
+        36754.0,
+        36827.0,
+        36827.0,
+        37086.0,
+        37086.0,
+        37153.0,
+        37280.0,
+        37717.0,
+        38720.0,
+        38890.0,
+        39157.0,
+        42931.0,
+        43182.0,
+        43436.0,
+        43653.0,
+        43833.0,
+        43833.0,
+        44056.0,
+        44250.0,
+        44310.0,
+        44483.0,
+        44739.0,
+        44845.0,
+        45075.0,
+        45075.0,
+        45267.0,
+        45267.0,
+        45469.0,
+        45469.0,
+        45686.0,
+        45686.0,
+        45960.0,
+        45960.0,
+        46179.0,
+        46179.0,
+        46394.0,
+        46394.0,
+        46695.0,
+        46829.0,
+        46829.0,
+        46920.0,
+        46920.0,
+        47125.0,
+        47325.0,
+        47325.0,
+        47613.0,
+        47613.0,
+        47888.0,
+        48023.0,
+        48279.0,
+        48462.0,
+        48795.0,
+        48988.0,
+        49204.0,
+        49597.0,
+        49697.0,
+        49793.0,
+        49876.0,
+        49876.0,
+        50011.0,
+        50132.0,
+        50132.0,
+        50421.0,
+        50854.0,
+        50854.0,
+        51094.0,
+        51094.0,
+        51292.0,
+        51471.0,
+        51705.0,
+        51705.0,
+        51705.0,
+        51988.0,
+        52129.0,
+        52220.0,
+        52512.0,
+        52736.0,
+        53017.0,
+        53116.0,
+        53239.0,
+        53239.0,
+        53634.0,
+        53920.0,
+        54115.0,
+        54321.0,
+        54576.0,
+        54673.0,
+        54948.0,
+        55258.0,
+        55258.0,
+        55451.0,
+        55682.0,
+        55897.0,
+        56190.0,
+        56412.0,
+        56611.0,
+        56964.0,
+        57116.0,
+        57435.0,
+        57548.0,
+        57669.0,
+        57873.0,
+        58172.0,
+        58338.0,
+        58404.0,
+        58583.0,
+        58739.0,
+        58981.0,
+        59199.0,
+        59484.0,
+        59706.0,
+        59902.0,
+        60160.0
+    ],
     "fixations": [
+        [
```

Comment:
First fixation repeats twice.  Can you please double check such issues in all files?